src/app/qgisapp.cpp

```diff
@@ -14096,11 +14096,21 @@ void QgisApp::populateProjectStorageMenu( QMenu *menu, bool saving )
           }
           else
           {
-            QMessageBox::critical( this,
-                                   tr( "Unable to save project %1" ).arg( uri ),
-                                   QgsProject::instance()->error(),
-                                   QMessageBox::Ok,
-                                   Qt::NoButton );
+            QMessageBox msgbox;
+
+            msgbox.setWindowTitle( tr( "Unable to save project %1" ).arg( uri ) );
```

Comment:
HIG convention is that message box title doesn't state the error, just the action. I.e. this should be "Save Project"